test/expected/alternate_users.out

```diff
@@ -182,13 +169,13 @@ SELECT * FROM _timescaledb_catalog.chunk_index ORDER BY hypertable_id;
 \set ON_ERROR_STOP 0
 INSERT INTO "customSchema"."Hypertable_1"(time, "Device_id", temp_c, humidity, sensor_1, sensor_2, sensor_3, sensor_4)
 VALUES(1257894000000000000, 'dev1', 31, 71, 72, 4, 1, 102);
-psql:include/ddl_ops_1.sql:56: ERROR:  duplicate key value violates unique constraint "_hyper_4_6_chunk_Unique1"
+psql:include/ddl_ops_1.sql:52: ERROR:  duplicate key value violates unique constraint "_hyper_4_6_chunk_Unique1"
```

Comment:
This is the bane of including files. Is there a way we can fix this by not having it print line numbers of included files?